server/events/command_runner.go

```diff
@@ -446,7 +446,7 @@ func (c *DefaultCommandRunner) deletePlans(ctx *CommandContext) {
 
 func (c *DefaultCommandRunner) updateDB(ctx *CommandContext, pull models.PullRequest, results []models.ProjectResult) (models.PullStatus, error) {
 	// Filter out results that errored due to the directory not existing. We
-	// don't store these in the database because they would never be "applyable"
+	// don't store these in the database because they would never be "applied"
```

Comment:
```suggestion
	// don't store these in the database because they would never be "applyable"
```

Reply:
this is intentional

Reply:
Google shows that `apply-able` is ~4x more common than `applyable` (which also triggers a "spelling suggestion") if you're amenable to that spelling.

Reply:
sure!

Reply:
b7349b9